src/config.ml

```diff
@@ -1664,6 +1709,7 @@ let a_config_entry : line A.t =
          a_dhcp_option;
          a_proto;
          a_proto_force;
+         a_protocol_flags <?> "protocol-flags";
```

Comment:
from what I remember, shouldn't we put the most eager at front -- i.e. don't let `a_proto` consume the "proto", but put the `a_protocol_flags` further above so it can try to consume the "protocol-flags"?